platform-sdk/consensus-metrics/src/main/java/module-info.java

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+module org.hiero.consensus.metrics {
+    exports org.hiero.consensus.metrics;
+    exports org.hiero.consensus.metrics.config;
+    exports org.hiero.consensus.metrics.extensions;
+    exports org.hiero.consensus.metrics.noop;
+    exports org.hiero.consensus.metrics.noop.internal;
```

Comment:
Nit: Again, this looks suspicious and should be investigated in the future. Hopefully, it is just required by tests.